src/main/java/org/example/service/PageMapService.java

```diff
@@ -0,0 +1,48 @@
+package org.example.service;
+
+import ch.qos.logback.classic.html.UrlCssBuilder;
+import org.example.constant.RequestArguments;
+import org.example.controller.DeveloperController;
+import org.example.controller.MainController;
+import org.example.model.Arguments;
+import org.example.model.Response;
+import org.example.constant.UrlPattern;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+public class PageMapService {
```

Comment:
Move `PageMapService` and `MainServlet` classes into separate package. (`web`)
Rename `PageMapService` to `PathMapper`

Reply:
fixed